test/Polly.Specs/PolicyTResultAsyncSpecs.cs

```diff
@@ -110,18 +110,6 @@ public async Task Executing_the_policy_function_should_throw_when_context_is_nul
         ex.And.ParamName.Should().Be("context");
     }
 
-    [Fact]
-    public async Task Execute_and_capturing_the_policy_function_should_throw_when_context_data_is_null()
-    {
-        var policy = Policy
-            .HandleResult(ResultPrimitive.Fault)
-            .RetryAsync((_, _, _) => { });
-
-        var ex = await policy.Awaiting(p => p.ExecuteAndCaptureAsync(_ => Task.FromResult(ResultPrimitive.Good), null!))
-              .Should().ThrowAsync<ArgumentNullException>();
-        ex.And.ParamName.Should().Be("context");
-    }
```

Comment:
This isn't a duplicate of the method above as it's calling `ExecuteAndCaptureAsync()` not `ExecuteAsync()`.

Reply:
The duplicates are not next to each other 
The deleted one
`public async Task Execute_and_capturing_the_policy_function_should_throw_when_context_data_is_null()`
duplicates this one
`public async Task Execute_and_capturing_the_policy_function_should_throw_when_context_is_null()`